webhook.go

```diff
@@ -35,6 +36,7 @@ type WebhookParams struct {
 	Components      []MessageComponent      `json:"components"`
 	Embeds          []*MessageEmbed         `json:"embeds,omitempty"`
 	AllowedMentions *MessageAllowedMentions `json:"allowed_mentions,omitempty"`
+	ThreadID        string                  `json:"thread_id,omitempty"`
```

Comment:
Given that the thread ID isn't part of the body, we should just have `` `json:"-"` `` here.